Quaternion/Quaternion.py

```diff
@@ -80,20 +83,58 @@ class Quat(object):
       >>> (q2 * q1).equatorial
       array([ 353.37684725,   34.98868888,   47.499696  ])
 
-     Be aware that for Chandra operations the correct formalism for applying a
-     delta quaternion ``dq`` to maneuver from ``q1`` to ``q2`` is::
-
-       >>> q2 = q1 * dq
-       >>> dq = q1.inv() * q2
-
-     The quaternion returned by ``q1 / q2`` using the divide operator represents the
-     delta quaternion in the INERTIAL FRAME, which is not what is used by the
-     spacecraft to represent maneuvers.
-
-     Instead use the ``dq`` method (or equivalently ``q1.inv() * q2``) for computing a
-     delta quaternion to maneuver from ``q1`` to ``q2``::
-
-       >>> dq = q1.dq(q2)
+    Note that each step is as described in the section :ref:`Equatorial -> Matrix <equatorialmatrix>`
+
+      >>> q1 = Quat((20, 0, 0))
```

Comment:
Should these be updated with the new keyword too?

Reply:
Done.